packages/v4/patternfly-docs/content/design-guidelines/components/drawer/drawer.md

```diff
@@ -18,4 +18,4 @@ A **splitter** allows you to create a layout with resizable panes. The orientati
 ### When to use a splitter
 Add a splitter to a drawer if you need to resize the width or height of a panel to give content more space. If data shown in a drawer has enough space, then you don’t need to use a splitter.
 
-<img src="./img/splitter-drawer.png" width="1500" />
+<img src="./img/splitter-drawer.png" alt="Splitter Drawer Elements" width="1500" />
```

Comment:
This alt text could be tweaked slightly, to something more along the lines of "Drawer with a splitter".